java-checks/src/main/java/org/sonar/java/checks/CheckList.java

```diff
@@ -20,66 +20,23 @@
-import org.sonar.java.checks.naming.MethodNameSameAsClassCheck;
-import org.sonar.java.checks.naming.MethodNamedEqualsCheck;
-import org.sonar.java.checks.naming.MethodNamedHashcodeOrEqualCheck;
-import org.sonar.java.checks.serialization.CustomSerializationMethodCheck;
-import org.sonar.java.checks.serialization.ExternalizableClassConstructorCheck;
-import org.sonar.java.checks.serialization.PrivateReadResolveCheck;
-import org.sonar.java.checks.serialization.SerialVersionUidCheck;
-import org.sonar.java.checks.serialization.SerializableComparatorCheck;
-import org.sonar.java.checks.serialization.SerializableFieldInSerializableClassCheck;
-import org.sonar.java.checks.serialization.SerializableObjectInSessionCheck;
-import org.sonar.java.checks.serialization.SerializableSuperConstructorCheck;
-import org.sonar.java.checks.unused.UnusedLabelCheck;
-import org.sonar.java.checks.unused.UnusedLocalVariableCheck;
-import org.sonar.java.checks.unused.UnusedMethodParameterCheck;
-import org.sonar.java.checks.unused.UnusedPrivateFieldCheck;
-import org.sonar.java.checks.unused.UnusedPrivateMethodCheck;
-import org.sonar.java.checks.unused.UnusedReturnedDataCheck;
-import org.sonar.java.checks.unused.UnusedTestRuleCheck;
-import org.sonar.java.checks.unused.UnusedTypeParameterCheck;
+
+import java.util.List;
+
+import org.sonar.java.checks.naming.*;
+import org.sonar.java.checks.serialization.*;
+import org.sonar.java.checks.unused.*;
 import org.sonar.java.checks.xml.ejb.DefaultInterceptorsLocationCheck;
 import org.sonar.java.checks.xml.ejb.InterceptorExclusionsCheck;
-import org.sonar.java.checks.xml.maven.ArtifactIdNamingConventionCheck;
-import org.sonar.java.checks.xml.maven.DependencyWithSystemScopeCheck;
-import org.sonar.java.checks.xml.maven.DeprecatedPomPropertiesCheck;
-import org.sonar.java.checks.xml.maven.DisallowedDependenciesCheck;
-import org.sonar.java.checks.xml.maven.GroupIdNamingConventionCheck;
-import org.sonar.java.checks.xml.maven.PomElementOrderCheck;
+import org.sonar.java.checks.xml.maven.*;
 import org.sonar.java.checks.xml.spring.SingleConnectionFactoryCheck;
 import org.sonar.java.checks.xml.struts.ActionNumberCheck;
 import org.sonar.java.checks.xml.struts.FormNameDuplicationCheck;
 import org.sonar.java.checks.xml.web.SecurityConstraintsInWebXmlCheck;
 import org.sonar.java.checks.xml.web.ValidationFiltersCheck;
-import org.sonar.java.se.checks.ConditionAlwaysTrueOrFalseCheck;
-import org.sonar.java.se.checks.CustomUnclosedResourcesCheck;
-import org.sonar.java.se.checks.DivisionByZeroCheck;
-import org.sonar.java.se.checks.LocksNotUnlockedCheck;
-import org.sonar.java.se.checks.NoWayOutLoopCheck;
-import org.sonar.java.se.checks.NonNullSetToNullCheck;
-import org.sonar.java.se.checks.NullDereferenceCheck;
-import org.sonar.java.se.checks.OptionalGetBeforeIsPresentCheck;
-import org.sonar.java.se.checks.UnclosedResourcesCheck;
+import org.sonar.java.se.checks.*;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Explicitly import the specific classes needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2208)